x-pack/plugins/alerting/server/saved_objects/index.ts

```diff
@@ -53,10 +53,28 @@ export const RuleAttributesExcludedFromAAD = [
   'running',
 ];
 
+export const RuleAttributesIncludedInAAD = [
+  'enabled',
+  'name',
+  'tags',
+  'alertTypeId',
+  'consumer',
+  'legacyId',
+  'schedule',
+  'actions',
+  'params',
+  'mapped_params',
+  'createdBy',
+  'createdAt',
+  'apiKeyOwner',
+  'apiKeyCreatedByUser',
+  'throttle',
+  'notifyWhen',
+  'meta',
+];
+
 // useful for Pick<RawAlert, RuleAttributesExcludedFromAAD> which is a
 // type which is a subset of RawAlert with just attributes excluded from AAD
-
-// useful for Pick<RawAlert, RuleAttributesExcludedFromAAD>
 export type RuleAttributesExcludedFromAADType =
```

Comment:
This is used in `partially_update_rule.ts`. Would it make more sense to update the logic there to check the include list so that we can entirely remove any exclude list definitions?